### PR TITLE
Fixed typo; add clarifications to functionalities

### PR DESCRIPTION
**Pull request by shunchu**
* Some typo and syntax updates
* Add clarification on the usage of each method for busy/careless coders like myself 😬
* Add an example `link` for the `#shorten_parameters` method to make the intent of the method more obvious

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -99,9 +102,11 @@ Or install it yourself as:
 ```
 
 ### Shorten parameters
+This method allows shortening of an URL with multiple parameters.
```
Comment (catsofmath) [inline README.md:105]: Very nice!!!

**Review by catsofmath — APPROVED** [review]
Looks super useful to me @shunchu ! I was totally confused by this as well 

@saiqulhaq thanks for work on this gem btw! Do you think update can/should get in : )

**Review by saiqulhaq — APPROVED** [review]

**Comment by saiqulhaq** [comment]
Thanks for fixing any typos and clearing up the usage

**Comment by catsofmath** [comment]
@saiqulhaq ty! : )

**Pull request merged**